src/server/mgflask/mgflask/models.py

```diff
@@ -123,14 +123,15 @@ def serialize_response(self):
             'right_bias': self.right_bias,
             'left_bias': self.left_bias,
             'content': self.content,
+            'replies': [reply.serialize_response for reply in self.replies],
             'comment_ratings': [rating.serialize_response for rating in self.comment_ratings],
         }
 
 class Reply(Base):
     __tablename__ = 'reply'
     id = Column(Integer, primary_key=True)
     comment_id = Column(Integer, ForeignKey('comment.id'), nullable=False)
-    comment = relationship('Comment')
+    comment = relationship('Comment', back_populates="replies")
```

Comment:
why do we need this?

Reply:
because replies are to comments as comments are to articles